tests/test_custom_checkers/test_redundant_assignment_checker.py

```diff
@@ -244,3 +244,20 @@ def test_augassign_redundant(self):
             ignore_position=True,
         ):
             self.checker.visit_augassign(augassign_node)
+
+    def test_annassign_redundant(self):
+        src = """
+        y_pos = 5
```

Comment:
I'd prefer if you removed this line, as it's not necessary for the test (unlike in the `augassign` test, where the initial assignment is necessary for the second line to be valid code).